plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/TSubjectAssumingTransport.java

```diff
@@ -42,15 +39,15 @@ public void open()
             throws TTransportException
     {
         try {
-            Subject.doAs(subject, (PrivilegedExceptionAction<?>) () -> {
+            Subject.callAs(subject, () -> {
                 transport.open();
                 return null;
             });
         }
-        catch (PrivilegedActionException e) {
+        catch (RuntimeException e) {
```

Comment:
Why not `CompletionException`? `NullPointerException` won't be thrown because the 2nd argument is always non-null.